src/app/modules/main/module.nim

```diff
@@ -598,4 +602,14 @@ method resolvedENS*[T](self: Module[T], publicKey: string, address: string, uuid
 
 method contactUpdated*[T](self: Module[T], publicKey: string) =
   let (name, image, isIdenticon) = self.controller.getContactNameAndImage(publicKey)
-  self.view.activeSection().updateMember(publicKey, name, image, isIdenticon)
\ No newline at end of file
+  self.view.activeSection().updateMember(publicKey, name, image, isIdenticon)
+
+method calculateProfileSectionHasNotification*[T](self: Module[T]): bool =
+  return not self.controller.isMnemonicBackedUp()
+
+method mnemonicBackedUp*[T](self: Module[T]) =
+  var profileItem = self.view.model().getItemById(conf.SETTINGS_SECTION_ID)
+  # I had to use this ugly syntax because `profileItem.hasNotification= ...` didn't compile
+  # This seems like a compilator error
+  `hasNotification=`(profileItem, self.calculateProfileSectionHasNotification())
```

Comment:
@jrainville I agree it looks a bit ugly :) and not sure why this doesn't work, maybe cause `SectionItem` is not a `ref object`, not sure, but anyhow let's make it a bit nicer using methods we already have, if you don't mind just update this `mnemonicBackedUp ` method to this:
```
method mnemonicBackedUp*[T](self: Module[T]) =
  self.view.model().udpateNotifications(conf.SETTINGS_SECTION_ID, self.calculateProfileSectionHasNotification(), 0)
```

Reply:
Good improvement. It worked and I updated